pkg/app/piped/eventwatcher/eventwatcher.go

```diff
@@ -33,6 +33,7 @@ import (
 	"go.uber.org/zap"
 	"google.golang.org/grpc"
 
+	"github.com/google/uuid"
```

Comment:
nit, for convention, should add this line to the above block

Reply:
thank you!
I fixed it. [rearranged package](https://github.com/pipe-cd/pipecd/pull/4395/commits/239600e0b4527ee907f4ecca013460217677e260)